core/modules/replica/DatabaseServicesMySQL.cc

```diff
@@ -1,6 +1,6 @@
 /*
  * LSST Data Management System
- * Copyright 2017 LSST Corporation.
+ * Copyright 2019 LSST Corporation.
```

Comment:
are you correcting the date or updating it? ...if updating, I thought we were supposed to give a range of years, e.g. 2017-2019

Reply:
Good point. Fixed